dair_pll/experiment.py

```diff
@@ -102,6 +102,8 @@ class SupervisedLearningExperimentConfig:
     """How many epochs should pass between full evaluations."""
     full_evaluation_samples: int = 5
     """How many trajectories to save in full for experiment's summary."""
+    gen_videos: bool = False
```

Comment:
`SupervisedLearningExperimentConfig` is more abstract than drake-related, so this likely should be stored somewhere else, if anywhere; it doesn't appear to be used in this PR.

Reply:
Noted.  Removed from this PR since it is unused, but I will take the advice moving forward.